tests/scripts/thread-cert/border_router/test_multi_ail.py

```diff
@@ -124,6 +124,25 @@ def test_multi_backbone_infra(self):
         self.assertTrue(br2.ping(br3_onlink_ula, backbone=True))
         self.assertTrue(br3.ping(br2_onlink_ula, backbone=True))
 
+    def test_adjcent_peer_br_count_less_than_netdata_peer_br_count(self):
+        """This test ensures that the adjcent_peer_br count is less than netdata_peer_br count in Mutil-AIL topology."""
```

Comment:
> ensures that the adjcent_peer_br count is less than netdata_peer_br count

But there isn't such a check in the test body. Maybe reword this description.

Reply:
Thanks, will update.